internal/handlers/blocker.go

```diff
@@ -32,9 +33,21 @@ func NewBlocker(logger *zap.Logger) *Blocker {
 
 func (b Blocker) BlockUserFunc(repository domain.BlockerRepository) users.BlockUserHandlerFunc {
 	return func(u users.BlockUserParams, access interface{}) middleware.Responder {
+		isManager, err := authentication.IsManager(access)
+		if err != nil {
+			b.logger.Error("error while getting authorization", zap.Error(err))
+			return users.NewDeleteUserDefault(http.StatusInternalServerError).
```

Comment:
Wouldn't be clearer if use `users.NewBlockUserDefault` instead of `users.NewDeleteUserDefault`?